### ISSUE TITLE
[4.0] Boolean "false" being translated to "null"

### ISSUE THREAD

**Issue by rafael-renan-pacheco** [yajra/laravel-oci8#122]
Hello!

After updating from 3.0 to 4.0, my seed scripts stopped working when setting "false" to "boolean" columns that does not allow null.

For example, I have the following table:

```php
Schema::create('perfil', function (Blueprint $table) {
    $table->increments('id')->start(2)->nocache();
    $table->string('nome', 30);
    $table->boolean('permanente'); /* This column does not allow null */
    $table->timestamps();
});
```

Then, when trying to seed it by running:
```php
$perfil = new Perfil;
$perfil->nome = 'Vendedor';
$perfil->permanente = false; /* In 3.0 this were translated to "0" */
$perfil->save();
```

I got the following error:
```
[Yajra\Pdo\Oci8\Exceptions\Oci8Exception]
Error Code    : 1400
Error Message : ORA-01400: cannot insert NULL into ("APP_SGC"."PERFIL"."PERMANENTE")
Position      : 76
Statement     : insert into perfil (nome, permanente, updated_at, created_at) values (:p0, :p1, :p2, :p3) returning id into :p4
Bindings      : [Vendedor,,2015-12-21 14:26:37,2015-12-21 14:26:37,0]
```

As you can see, `:p1`, which is `$perfil->permanente = false` got translated to `null` instead of `0`.

Is this a desired behavior in 4.0, since in 3.0 it worked just fine?

Best regards,
Rafael Pacheco.

**Comment by yajra** [comment]
@rafael-renan-pacheco, thanks for reporting. Will check this as soon as I can.

**Comment by yajra** [comment]
I just tested this and it works fine for me. There might be some mutators on your model that causes it to null?

```php
get('admin/test', function() {
    $model         = new Permission();
    $model->slug   = 'slug';
    $model->name   = 'name';
    $model->system = false;
    $model->save();

    return $model;
});

**Comment by yajra** [comment]
`$table->boolean('permanente'); /* This column does not allow null */`.

I was able to reproduce the bug. My boolean is nullable.

**Comment by yajra** [comment]
@rafael-renan-pacheco, this should be fixed on v4.0.1. Kindly check again and test. Thanks!

**Comment by rafael-renan-pacheco** [comment]
Thank you very much!

After updating, the seed script worked with "false" again.

Best regards,
Rafael.
